docs/analytics.rst

```diff
@@ -68,17 +68,21 @@ For all generic CTA links and <button> elements, add these data attributes (* in
 |    data-cta-position     | Location of CTA on the page (e.g. 'primary', 'secondary', 'header') |
 +--------------------------+---------------------------------------------------------------------+
 
-For all download buttons, add these data attributes:
-
-+--------------------------+--------------------------------+
-|    Data Attribute        |        Expected Value          |
-+==========================+================================+
-|    data-link-type        | 'Desktop', 'Android', or 'iOS' |
-+--------------------------+--------------------------------+
-|    data-download-os      | name or text of the link       |
-+--------------------------+--------------------------------+
-|    data-download-version |'standard', 'developer', 'beta' |
-+--------------------------+--------------------------------+
+For all download buttons, add these data attributes (* indicates a required attribute):
+
++---------------------------+-----------------------------------------------------------------------------------------------+
+|    Data Attribute         |        Expected Value                                                                         |
++===========================+===============================================================================================+
+|    data-link-type *       | 'download'                                                                                    |
++---------------------------+-----------------------------------------------------------------------------------------------+
+|    data-download-os *     | 'Desktop', 'Android', 'iOS'                                                                   |
++---------------------------+-----------------------------------------------------------------------------------------------+
+|    data-download-name     | 'Windows 32-bit', 'Windows 64-bit', 'macOS', 'Linux 32-bit', 'Linux 64-bit', 'iOS', 'Android' |
++---------------------------+-----------------------------------------------------------------------------------------------+
+|    data-download-version  | 'win', 'win64', 'osx', 'linux', 'linux64', 'ios', 'android'                                   |
```

Comment:
> (* indicates a required attribute)

Would the download button still be functional without `data-download-version`? Or should we reword "required attributes for analytics"?

Reply:
As per Justin's comment here, only `data-link-type` and `data-download-os` are required attributes. Attributes such as `data-download-version` are not required for th goal (there's also only one bersion for iOS and Android, which kinda makes it redundant). So I think we're good here :)

Reply:
After thinking about this a bit more, I agree it's worth adding a note for clarity - which I did here: https://gitlab.com/mozmeao/bedrock/-/commit/853f7bee77e7f993b5a7a2120d78e52d0bcad414 👍